parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -386,7 +384,7 @@ impl ExportXcm for BridgeHubRococoOrBridgeHubWococoSwitchExporter {
 					destination,
 					message,
 				)
-				.map(|result| ((EthereumNetwork::get(), result.0), result.1))
+				.map(|result| ((EthereumNetwork::get(), (result.0, XcmHash::default())), result.1))
```

Comment:
Because `SnowbridgeExporter::validate` no longer returns the `XcmHash`. Not sure what the solution is here, I think @vgeddes mentioned in a newer version of Substrate the other validators also do not return the XcmHash?

Reply:
Yeah returning `XcmHash::default` is fine here.

When we upgrade to the latest cumulus it will all be fine.